modules/nw-modifying-operator-install-config.adoc

```diff
@@ -46,6 +46,19 @@ $ ./openshift-install create manifests --dir=<installation_directory> <1>
 <1> For `<installation_directory>`, specify the name of the directory that
 contains the `install-config.yaml` file for your cluster.
 
+. Modify the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines:
++
+--
+.. Open the `manifests/cluster-scheduler-02-config.yml` file.
+.. Locate the `mastersSchedulable` parameter and set its value to `False`.
+.. Save and exit the file.
+--
++
+[NOTE]
+====
+Currently, due to a link:https://github.com/kubernetes/kubernetes/issues/65618[Kubernetes limitation], router Pods running on control plane machines will not be reachable by the ingress load balancer. This step might not be required in a future minor version of {product-title}.
```

Comment:
Recommend against saying `This step might not be required in a future minor version of {product-title}.` If it's no longer required in a future release, we should just make an edit to remove the step.